server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/copybooks/analysis/CopybookName.java

```diff
@@ -38,11 +38,11 @@ public CopybookName(String displayName, String dialectType) {
   }
 
   /**
-   * Creates a processing copybook name using a name and a dialect type
-   *
+   * Creates a processing copybook name (cache id) using a name and a dialect type
+   * @param documentUri Document with COPY instruction for this copybook.
    * @return a processing copybook name
    */
-  public String getProcessingName() {
-    return String.format("%s#%s", qualifiedName, dialectType);
+  public String getProcessingName(String documentUri) {
```

Comment:
Why `documentUri` is not a part of this `CopybookName`?

Reply:
CopybookName can be called from different documents, so it's a parameter for a cache key. (moved to cache service now)